src/main/kotlin/no/nav/medlemskap/domene/Dokument.kt

```diff
@@ -17,16 +19,37 @@ data class Journalpost(
     val journalstatus: String?,
     val tema: String?,
     val sak: Sak?,
-    val dokumenter: List<Dokument>?
+    val dokumenter: List<Dokument>?,
 ) {
+
     companion object {
         private val tillatteTemaer = listOf("MED", "UFM", "TRY")
 
+        fun Journalpost.erMindreEnn1AarGammelt(): Boolean {
+            val today = LocalDate.now()
+            val reldato = this.relevanteDatoer?.find { it.datotype == Datotype.DATO_JOURNALFOERT }
+            if (reldato != null) {
+                val date = LocalDate.parse(reldato.dato, DateTimeFormatter.ISO_LOCAL_DATE_TIME)
+                return date.isAfter(today.minusYears(1))
```

Comment:
Er det riktig å bruke today og ikke første dag for søknaden? Hva om bruker søker om sykepenger 1 år etter at de har vært syk?